### PR TITLE
Bump dependencies to dioxus 0.6.

### PR DESCRIPTION
**Pull request by Emm**

### CODE REVIEW

File: Cargo.toml
Comment (marc2332) [inline Cargo.toml:1]: Any reason you use git links on this one? Rust crates can't be released with git dependencies

**Review by marc2332 — CHANGES_REQUESTED** [review]

File: Cargo.toml [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
 
 [dependencies]
-dioxus = { version = "0.6.0-alpha.3", features = ["web"] }
-dioxus-resize-observer = "0.3.0-alpha.4"
-dioxus-use-mounted = "0.3.0-alpha.4"
+dioxus = { version = "0.6.0", features = ["web"] }
+dioxus-resize-observer = { git = "https://github.com/Emm/dioxus-resize-observer.git" }
+dioxus-use-mounted = { git = "https://github.com/dioxus-community/dioxus-use-mounted" }
 log = "0.4.22"
 dioxus-logger = "0.5.0"
```
Comment (marc2332) [inline Cargo.toml:14]: Could you also update this one to 0.6.0?

**Comment by Emm** [comment]
Sorry, I had forgotten to git push -f, should have re-read before opening
the PR

File: Cargo.toml (hunk 1 above)
Comment (marc2332) [reply Cargo.toml:14]: Not sure if you misses this so here goes this comment :)

File: Cargo.toml
Comment (Emm) [reply Cargo.toml:1]: By mistake :)

**Review by marc2332 — APPROVED** [review]

**Comment by marc2332** [comment]
Thanks!!

**Comment by Emm** [comment]
Thanks for being so responsive :)

**Pull request merged**